src/iop/channelmixerrgb.c

```diff
@@ -1915,6 +1915,9 @@ void gui_init(struct dt_iop_module_t *self)
   NOTEBOOK_PAGE(grey, grey, N_("grey"), N_("grey"), FALSE)
 
   self->widget = GTK_WIDGET(g->notebook);
+  int active_page = dt_conf_get_int("plugins/darkroom/channelmixerrgb/gui_page");
```

Comment:
const

Reply:
ok. I copied this from tone equalizer, so have added a const there as well.